vms/event/tests/test_shiftSignUp.py

```diff
@@ -215,3 +215,120 @@ def test_empty_events(self):
+
+        # Create another volunteer
+        volunteer_user_2 = User.objects.create_user(
+                username = 'volunteer-2',
+                password = 'volunteer')
+
+        volunteer_2 = Volunteer.objects.create(
+                user = volunteer_user_2,
+                address = 'address',
+                city = 'city',
+                state = 'state',
+                country = 'country',
+                phone_number = '9999999999',
+                email = 'volunteer2@volunteer.com',
+                unlisted_organization = 'organization')
+
+        # Assign shift to the volunteer
+        VolunteerShift.objects.create(
+                shift = shift_2,
+                volunteer = volunteer_2)
+
+        # Login as volunteer 1 and open Shift Sign Up
+        self.login()
+        self.driver.find_element_by_link_text('Shift Sign Up').click()
+
+        # on event page
+        self.assertEqual(self.driver.find_element_by_class_name('alert-info').text,'There are no events.')
+
+    def test_search_event(self):
+
+        self.register_event_utility()
+        self.register_job_utility()
+        self.register_shift_utility()
+
+        # Login as volunteer 1 and open Shift Sign Up
+        self.login()
+        self.driver.find_element_by_link_text('Shift Sign Up').click()
+
+        # enter date range in which an event starts
+        self.driver.find_element_by_id('from').send_keys('05/08/2016')
+        self.driver.find_element_by_id('to').send_keys('08/31/2017')
+        self.driver.find_element_by_xpath('//form[1]').submit()
+        # verify that the event shows up
+        self.assertEqual(self.driver.find_element_by_xpath('//table//tbody//tr[1]//td[1]').text, 'event')
+
+        # enter date range in which no event starts
+        self.driver.find_element_by_id('from').clear()
+        self.driver.find_element_by_id('to').clear()
+        self.driver.find_element_by_id('from').send_keys('10/08/2016')
+        self.driver.find_element_by_id('to').send_keys('08/31/2017')
+        self.driver.find_element_by_xpath('//form[1]').submit()
+        # verify that no event shows up on event page
+        self.assertEqual(self.driver.find_element_by_class_name('alert-info').text,'There are no events.')
+
+        # enter only incorrect starting date
+        self.driver.find_element_by_id('from').clear()
+        self.driver.find_element_by_id('to').clear()
+        self.driver.find_element_by_id('from').send_keys('10/08/2016')
+        # verify that no event shows up on event page
+        self.assertEqual(self.driver.find_element_by_class_name('alert-info').text,'There are no events.')
+
+        """# enter only correct starting date
+        self.driver.find_element_by_id('from').clear()
+        self.driver.find_element_by_id('from').send_keys('05/10/2016')
+        # verify that the event shows up
+        self.assertEqual(self.driver.find_element_by_xpath('//table//tbody//tr[1]//td[1]').text, 'event')"""
```

Comment:
@tapasweni-pathak Yes, these tests verify the correct behavior of search event form. I have commented them out because the tests fail due to bug #337 at this point. Once the bug is fixed, these can be uncommented.